### PR TITLE
Feat/370031  cdo expiry should default

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/repos/exemption.js [hunk 1]
```diff
@@ -122,9 +122,9 @@ const handleOrder2023 = (registration, data) => {
  */
 const setDefaults = (registration, data) => {
   if (
-    (registration.cdo_expiry === null || registration.cdo_expiry === undefined) &&
-    (registration.cdo_issued === null || registration.cdo_issued === undefined) &&
-    (data.cdoIssued !== null && data.cdoIssued !== undefined)
+    (data.cdoExpiry === null || data.cdoExpiry === undefined) &&
```
Comment (jbarnsley10) [inline app/repos/exemption.js:125]: Is it not easy to just write 
if (!data.cdoExpiry && !data.cdoIssued & etc
??

File: app/repos/exemption.js [hunk 2]
```diff
@@ -122,9 +122,9 @@ const handleOrder2023 = (registration, data) => {
...
+    (data.cdoIssued !== null && data.cdoIssued !== undefined) &&
+    (registration.previous('cdo_issued') === null || registration.previous('cdo_issued') === undefined)
```
Comment (jbarnsley10) [inline app/repos/exemption.js:127]: Yes, the previous() is a better way to get previous values, but we already have them at the beginning of this call. Happy to let this go but it does make this logic sequelize-specific i.e. if we changed the ORM we would have to identify the change necessary in the logic (not just the queries).

**Review by jbarnsley10 — APPROVED** [review]
I'm happy to approve but decide if you want to change the null/undefined

File: app/repos/exemption.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/repos/exemption.js:125]: was concerned here about it catching falsy values, prefer to be more explicit.  Not that I can foresee a falsy value being caught here.

**Review by jbarnsley10 — APPROVED** [review]

**Pull request merged**